Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.Shared.cs

```diff
@@ -296,13 +296,40 @@ public void ResolveAbsolutePositionedElement ()
 
     [Category("Screenshot")]
     [Test]
-    public void ResolveBorderElementA ()
+    [TestCase("Bottom", $"{nameof(ResolveBorderElement)}Bottom")]
+    [TestCase("Right", $"{nameof(ResolveBorderElement)}Right")]
+    public void ResolveBorderElement (string testSide, string testName)
+    {
+      // TODO: Remove when RM-7187 is resolved, use the RetryAttribute provided by NUnit instead.
+      RetryTest(
+          () =>
+          {
+            var home = Start<HtmlPageObject>("ScreenshotBorderTest.aspx");
+
+            // Chromium browsers show the remote control overlay since RM-7184, which is drawn shortly after the page load. The
+            // content however needs a small timespan to be adapted to the slightly smaller viewport.
+            if (Helper.BrowserConfiguration.IsChromium())
+              Thread.Sleep(100);
+
+            ScreenshotTestingDelegate<IFluentScreenshotElement<ElementScope>> test =
+                (builder, target) => { builder.Crop(target, new WebPadding(1)); };
+
+            var element = home.Scope.FindId($"borderElement{testSide}").ForElementScopeScreenshot();
+
+            Helper.RunScreenshotTestExact<IFluentScreenshotElement<ElementScope>, ScreenshotTest>(element, ScreenshotTestingType.Both, test, memberName: testName);
+          },
+          2);
+    }
+
+    [Category("Screenshot")]
+    [Test]
+    public void ResolveBorderElementLeft ()
```

Comment:
Should be done like ResolveborderElement in line 301, or, possibly, merged with the other test.
@Frazerus